tox.ini

```diff
@@ -22,7 +22,7 @@ commands =
     m8a-fetch-language-containers {envname}
     # -n auto tells pytest to use as many processors as detected
     # to run all the tests from the module test_{envname} in parallel
-    pytest -n auto -m "not serial" test_{envname}.py --junitxml={toxinidir}/junit_parallel.xml []
+    pytest -n auto -m "not serial" --suppress-no-test-exit-code test_{envname}.py --junitxml={toxinidir}/junit_parallel.xml []
```

Comment:
This can be removed, as all the tests in performance should be marked as serial (to run in the same host, same environment)

Reply:
Well, we should create another unit for that. Let me fix that now.